core/src/test/java/io/confluent/kafka/schemaregistry/RestApp.java

```diff
@@ -35,30 +35,30 @@
   public Server restServer;
   public String restConnect;
 
-  public RestApp(int port, String zkConnect, String kafkaTopic) {
+  public RestApp(Integer port, String zkConnect, String kafkaTopic) {
```

Comment:
Does this file need to change?  The `Integer` will get unboxed as an `int` when passed.